content/docs/operating/configuration.md

```diff
@@ -268,6 +272,48 @@ Consul setups, the relevant address is in `__meta_consul_service_address`.
 In those cases, you can use the [relabel](#target-relabeling-relabel_config)
 feature to replace the special `__address__` label.
 
+### Kubernetes SD configurations `<kubernetes_sd_config>`
+
+CAUTION: Kubernetes SD is in beta: breaking changes to configuration are possible
+in future releases.
+
+Kubernetes SD configurations allow retrieving scrape targets from
+[Kubernetes'](http://kubernetes.io/) REST API. By default, this discovers nodes
+& appropriately annotated services so that metrics from both cluster
```

Comment:
Haha, I could ask the reverse :) It's not common in formal English technical documentation to use "&" for "and" in sentences, and we have to settle on one style, so "and" it is.

http://www.quickanddirtytips.com/education/grammar/ampersand

"The ampersand (&) is a symbol;for and. Unlike the percent or degree symbol, you generally shouldn't use the ampersand except in the most informal situations. - See more at: http://www.quickanddirtytips.com/education/grammar/ampersand#sthash.vQxpuZyz.dpuf"